freqtrade/rpc/telegram.py

```diff
@@ -738,6 +739,50 @@ def _version(self, update: Update, context: CallbackContext) -> None:
         """
         self._send_msg('*Version:* `{}`'.format(__version__))
 
+    @authorized_only
+    def _stats(self, update: Update, context: CallbackContext) -> None:
+        """
+        Handler for /stats
+        https://github.com/freqtrade/freqtrade/issues/3783
+        Show stats of recent trades
+        :param update: message update
+        :return: None
+        """
+        # TODO: self._send_msg(...)
+        def trade_win_loss(trade):
+            if trade['profit_abs'] > 0:
+                return 'Wins'
+            elif trade['profit_abs'] < 0:
+                return 'Losses'
+            else:
+                return 'Draws'
+
+        trades = self._rpc_trade_history(-1)
+        trades_closed = [trade for trade in trades if not trade['is_open']]
+
+        # Sell reason
+        sell_reasons = {}
+        for trade in trades_closed:
+            if trade['sell_reason'] in sell_reasons:
+                sell_reasons[trade['sell_reason']][trade_win_loss(trade)] += 1
+            else:
+            win_loss_count = {'Wins': 0, 'Losses': 0, 'Draws': 0}
+            win_loss_count[trade_win_loss(trade)] += 1
+            sell_reasons[trade['sell_reason']] = win_loss_count
+        sell_reason_msg = [
+            '| Sell Reason | Sells | Wins | Draws | Losses |',
+            '|-------------|------:|-----:|------:|-------:|'
```

Comment:
You should be able to use `tabulate()` for this.

It'll do a much better job at formatting a table than we could... (we're using it for `_status_table()` for example).